src/main/resources/com/turo/pushy/console/pushy-console_de.properties

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2018 Turo Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+alert.bad-certificate.title=Ungültiges Zertifikat
+alert.bad-certificate.header=Das ausgewählte Zertifikat ist kein gültiges APNs Zertifikat.
+alert.bad-certificate.content-text=Die UID im Subject Feld des Zertiifikates ist kein gültiges 'APNs topic'.
+alert.notification-failed.title=Senden der Push-Benachrichtigung fehlgeschlagen
+alert.notification-failed.header=Beim Senden der Push-Benachrichtigung ist ein Fehler aufgetreten.
+certificate-chooser.filter.pkcs8_and_pkcs12=PKCS#8 und PKCS#12 Dateien
+certificate-chooser.filter.pkcs12=PKCS#12 Dateien (Zertifikate)
+certificate-chooser.filter.pkcs8=PKCS#8 Dateien (Authentifizierungs-Token)
+certificate-chooser.filter.all=Alle Dateien
+certificate-password-dialog.title=Kennwort eingeben
+certificate-password-dialog.header=Bitte geben Sie das Kennwort für {0} ein.
+certificate-password-dialog.prompt=Kennwort
+delivery-priority.immediate=Sofort
+delivery-priority.conserve-power=Stromsparend
+notification-result.placeholder=Keine Push-Benachrichtigung gesendet
+notification-result.details.accepted=n.a.
+notification-result.details.expiration={0} ({1,date,dd-MM-yyy} {1,time,HH:mm:ss})
+notification-result.status.accepted=Angenommen
+notification-result.status.rejected=Zurückgewiesen
+password-dialog.incorrect-password=Das eingegebene Kennwort ist nicht korrekt.
+pushy-console.title=Pushy Console
+fxml.apns-server.label=APNs Server
+fxml.port.label=Port
+fxml.credentials.label=Token / Zertifikat
+fxml.browse.label=Durchsuchen…
+fxml.key-id.label=Token ID / Zertifikats ID
+fxml.team-id.label=Team ID
+fxml.topic.label=Topic
+fxml.token.label=Device Token
```

Comment:
Huh… this is a straight cognate to German?

Reply:
I must admit that I could not find a proper translation for 'Topic'. Following the Apple Developer Account website 'Bundle Identifier' or 'App ID' might be better because Apple uses these. 

'Device Token' is used in german literature too. I think a formally correct translation might not be helpful in the sense what the user should enter in these fields.

Reply:
Cool. Thanks for clarifying!